### PR TITLE
remove cc alternative group

### PR DESCRIPTION
**Pull request by jnbr**
The alternative group for cc keeps causing problems, mostly with dkms.
I think the benefit of keeping it isn't worth the trouble it causes.

### CODE REVIEW

File: srcpkgs/gcc/template [hunk 1]
```diff
@@ -364,6 +363,8 @@ do_install() {
 	ln -sfr ${DESTDIR}/usr/include/c++/${_minorver} \
 		${DESTDIR}/usr/include/c++/${version}
 
+	# cc symlink
+	ln -sfr ${DESTDIR}/usr/bin/gcc ${DESTDIR}/usr/bin/cc
```
Comment (Chocimier) [inline srcpkgs/gcc/template]: target should be `/usr/bin/gcc` or `gcc`.

**Comment by Duncaen** [comment]
> Does it install fine when one has cc set to other than gcc?

The latest xbps release should handle that, it now checks if the file is a symlink before unlinking it.
Before it would overwrite the symlink with a file and then update the alternatives after unpacking the package and then remove all files that where previously alternatives.
This might still break files that where alternatives and are symlinks after the update, so it should be tested nonetheless.

**Comment by jnbr** [comment]
I tested two cases:
 - clang and gcc installed, alternative set to gcc
 - clang and gcc installed, alternative set to clang

In both cases, a full system upgrade worked as expected with cc pointing to gcc afterwards.

File: srcpkgs/gcc/template (hunk 1 above)
Comment (jnbr) [reply srcpkgs/gcc/template]: Please clarify, I don't see the issue.

Comment (Chocimier) [reply srcpkgs/gcc/template]: Sorry, i missed -r flag. It's fine.

**Comment by leahneukirchen** [comment]
+1 from me for removal, it confused some things here too.

**Comment by xtraeme** [comment]
So how many likes do we need to apply it?

**Comment by Chocimier** [comment]
all of them

**Comment by xtraeme** [comment]
ffs push this. This alternative is really stupid.

**Comment by jnbr** [comment]
After upgrading both, clang and gcc, ` xbps-alternatives -l` still shows the alternative group:

```
cc
 - clang (current)
 - gcc
```

but `xbps-alternatives -l gcc` or `xbps-alternatives -l clang` do not show it.


> ffs push this. This alternative is really stupid.

Relax, it has been like this for over a year.

**Pull request merged**